terminologies/handedness/ambidextrousHandedness.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@id": "https://openminds.ebrains.eu/controlledTerms/handedness/ambidextrousHandedness",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Handedness",
+  "name": "ambidextrous handedness",
+  "definition": "Handedness where the organism exhibits no overall dominance in the use of right or left hand or foot in the performance of tasks that require one hand or foot or a dominant hand or foot.",
```

Comment:
I would delete the last part "....or dominant hand or foot." (is a bit over defined for my taste)

Reply:
Agreed. How about a compromiss: "[...] require one (dominant) hand or foot."?